src/Polly.Shared/PolicyResult.cs

```diff
@@ -117,7 +143,13 @@ internal static PolicyResult<TResult> Failure(Exception exception, ExceptionType
                 context);
         }
 
-        internal static PolicyResult<TResult> Failure(TResult handledResult, Context context)
+        /// <summary>
+        /// Builds a failed <see cref="PolicyResult" />
```

Comment:
((suggest change to))

`Builds a <see cref="PolicyResult" /> representing a failed execution through the policy.`